build/rhea/configuration.nix

```diff
@@ -1,55 +1,28 @@
-{ pkgs, ... }:
 {
   imports = [
     ./hardware-configuration.nix
     ./hetzner.nix
     ./network.nix
     ../common.nix
-    ../hydra.nix
-    ../hydra-proxy.nix
-    ../hydra-scaler.nix
-    ../packet-importer.nix
   ];
 
   networking = {
     hostName = "rhea";
-    firewall.allowedTCPPorts = [
-      80
-      443
-      9198 # hydra-queue-runner's prometheus
-      9199 # hydra-notify's prometheus
-    ];
     firewall.allowPing = true;
-    firewall.logRefusedConnections = false;
   };
 
+  users.users.root.openssh.authorizedKeys.keys = [
+    "ssh-ed25519 AAAAC3NzaC1lZDI1NTE5AAAAIIY0EGPGfXD1L+YdSJIKUzeFyuUfVW58kMh+mSflEFx1 root@mimas"
+  ];
+
   system.stateVersion = "21.11";
 
-  services.hydra-dev.dbi = "dbi:Pg:dbname=hydra;host=10.254.1.9;user=hydra;";
   systemd.services.hydra-init = {
     after = [ "wireguard-wg0.service" ];
     requires = [ "wireguard-wg0.service" ];
   };
```

Comment:
This `hydra-init` dependency was dropped now?

Reply:
I'm actually not sure what to do with it, given that it didn't work in the first place.

Reply:
Oh, OK.  If it didn't work, I'd keep it dropped without replacement in this pull request.  (possibly some TODO comment)

Reply:
We already have #366 filed.